daffodil-lib/src/main/scala/org/apache/daffodil/lib/xml/QNameBase.scala

```diff
@@ -518,28 +521,25 @@ protected trait RefQNameFactoryBase[T] {
   def resolveRef(
```

Comment:
Suggest improve the scaladoc above because this code is handling both prefixed and non-prefixed refs.  The existing scaladoc suggests this is only about QNames with an explicit prefix.